cmd/nirvana/project/init.go

```diff
@@ -709,7 +762,10 @@ build:
 	    $(CMD_DIR)/$${target};                                                         \
 	done
 
-container:
+mod-reset-vendor:
+	@$(shell [ -f go.mod ] && go mod vendor)
```

Comment:
Have you tested this line?

In my case,  this command always output:
```
$ make mod-rest-vendor
make: 'mod-rest-vendor' is up to date.
```

Reply:
I've tested `@$(shell [ -f go.mod ] && echo 'something')`. But it alway outputs `make: 'mod-rest-vendor' is up to date.` even though I changed `go.mod`.

Reply:
On my computer:

![screen shot 2018-09-30 at 4 08 16 pm](https://user-images.githubusercontent.com/9097390/46255101-bf331000-c4cb-11e8-975e-b548b5bb60f3.png)

![screen shot 2018-09-30 at 4 09 00 pm](https://user-images.githubusercontent.com/9097390/46255328-3b2e5780-c4ce-11e8-8e87-b591da7ea773.png)

- - -

Would you like verbose logs as shown below ?

![screen shot 2018-09-30 at 4 09 33 pm](https://user-images.githubusercontent.com/9097390/46255344-63b65180-c4ce-11e8-8a39-376a05710a07.png)

Reply:
I've re-tested it. You are right.